source/cursorManager.py

```diff
@@ -313,6 +313,11 @@ def script_copyToClipboard(self,gesture):
 			# Translators: Message presented when text has been copied to clipboard.
 			ui.message(_("Copied to clipboard"))
 
+	def waitForAndReportSelectionChange(self, oldTextInfo):

```

Comment:
Considering that this implementation does not have to wait, perhaps this should be named differently?